manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/api/ManagerConstraintViolationExceptionMapper.java

```diff
@@ -0,0 +1,35 @@
+package com.redhat.service.smartevents.manager.v2.api;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+import javax.ws.rs.ext.Provider;
+
+import com.redhat.service.smartevents.infra.core.exceptions.BridgeErrorService;
+import com.redhat.service.smartevents.infra.core.exceptions.mappers.ConstraintViolationExceptionMapper;
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorResponse;
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+
+@Provider
+@ApplicationScoped
+public class ManagerConstraintViolationExceptionMapper extends ConstraintViolationExceptionMapper {
+
+    @Inject
+    public ManagerConstraintViolationExceptionMapper(BridgeErrorService bridgeErrorService) {
+        super(bridgeErrorService);
+    }
+
+    @Override
+    @PostConstruct
+    protected void init() {
+        super.init();
+    }
+
+    @Override
+    protected ErrorResponse toErrorResponse(ConstraintViolation<?> cv) {
+        ErrorResponse errorResponse = super.toErrorResponse(cv);
+        errorResponse.setHref(V2APIConstants.V2_ERROR_API_BASE_PATH + errorResponse.getId());
+        return errorResponse;
+    }
+}
```

Comment:
Does it work outside the manager-v2 package (for example in the final `manager` application that combines v1 and v2)?

Reply:
That is my fear.. that we'd have an `ExceptionMapper` for both `v1` and a `v2`. Both would map the same exceptions and we'd end up duplicating every mapped error.. Perhaps I'll remove this and edit the test until Errors has been redesigned.

Reply:
Removed until Error Handling refactor is complete.